app/facades/item_facade.rb

```diff
@@ -0,0 +1,29 @@
+class ItemFacade
+
+  def self.all_items
+    all_items = items_data[:data].map do |data|
+      ItemPoro.new(data)
+    end
+  end
+
+  def self.find_items(merchant_id)
+    items = all_items.select do |i|
+      i.merchant_id.to_s == merchant_id
+    end
+  end
```

Comment:
It would be more efficient to add the `api/v1/merchants/{:id}/items` endpoint to your service rather than getting all items and using ruby to iterate through them all to find those that match the merchant id. Since we know the merchant id, we can pass that to our service and get the results back directly from the api.